pom.xml

```diff
@@ -87,12 +87,12 @@
         <dependency>
             <groupId>com.sk89q.worldedit</groupId>
             <artifactId>worldedit-core</artifactId>
-            <version>7.0.0-20181118.055910-28</version>
+            <version>7.0.0-SNAPSHOT</version>
         </dependency>
         <dependency>
```

Comment:
Remove if not needed

Reply:
I guess I should've put a line comment after, not before since github shows lines above not below. In this one I was trying to mention the WorldGuard-core dependency.

Reply:
worldguard-legacy includes both worldedit-core and worldguard-core - at least this is what i get in intellij